packages/account/src/providers/provider.ts

```diff
@@ -1058,7 +1104,8 @@ Supported fuel-core version: ${supportedVersion}.`
    * transaction. The default value is 0.2 or 20%
    *
    * @param transactionRequestLike - The transaction request object.
-   * @param tolerance - The tolerance to add on top of the gasUsed.
+   * @param transactionCostParams - The transaction cost parameters (optional).
+   *
```

Comment:
Since tolerance was removed we should also remove:

```
   * @privateRemarks
   * The tolerance is add on top of the gasUsed calculated
   * from the node, this create a safe margin costs like
   * change states on transfer that don't occur on the dryRun
   * transaction. The default value is 0.2 or 20%
```

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2276/commits/5955ab1ebd70f04d9fd1661ee4199d2236c5970c